commit-event-service/src/test/scala/io/renku/commiteventservice/events/categories/common/CommitInfoFinderSpec.scala

```diff
@@ -267,5 +235,52 @@ class CommitInfoFinderSpec
       "committed_date":  "2012-09-20T09:06:12+03:00",
       "parent_ids":      ${parents.map(_.value)}
     }"""
+
+    val endpointName: String Refined NonEmpty = "commits"
```

Comment:
Wouldn't it be better to name this endpoint _commit-details_ or something which would be telling more explicitly that we are fetching a single commit info?